apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -676,20 +676,102 @@ Optional arguments:
 `;
 
 exports[`CommandLineHelp prints the help for each action: list 1`] = `
-"usage: rush list [-h] [-v] [-p] [-f] [--json]
+"usage: rush list [-h] [-v] [-p] [--full-path] [--json] [-t PROJECT]
+                 [-T PROJECT] [-f PROJECT] [-o PROJECT] [-i PROJECT]
+                 [-I PROJECT] [--to-version-policy VERSION_POLICY_NAME]
+                 [--from-version-policy VERSION_POLICY_NAME]
+                 
 
 List package names, and optionally version (--version) and path (--path) or 
 full path (--full-path), for projects in the current rush config.
 
 Optional arguments:
-  -h, --help       Show this help message and exit.
-  -v, --version    If this flag is specified, the project version will be 
-                   displayed in a column along with the package name.
-  -p, --path       If this flag is specified, the project path will be 
-                   displayed in a column along with the package name.
-  -f, --full-path  If this flag is specified, the project full path will be 
-                   displayed in a column along with the package name.
-  --json           If this flag is specified, output will be in JSON format.
+  -h, --help            Show this help message and exit.
+  -v, --version         If this flag is specified, the project version will 
+                        be displayed in a column along with the package name.
+  -p, --path            If this flag is specified, the project path will be 
+                        displayed in a column along with the package name.
+  --full-path           If this flag is specified, the project full path will 
```

Comment:
Maybe `-a` or `--absolute-path`? But this would be a bigger breaking change than the removal of `-f` so I'm not too concerned on this.

Reply:
My assumption is that no one should be using this in a script unless they are passing `--json`, in which case the `--full-path` flag doesn't actually do anything. Also, scripts should be using the full parameter name anyway. @octogonz?

Reply:
Specifying `-f` as a shorthand for `--full-path` seems unnecessary to me.

The shorthands should be introduced sparingly, for parameters that are very heavily used.  This is why the `ts-command-line` design intentionally restricts you to single letters.

> Also, scripts should be using the full parameter name anyway.

Agreed, the shorthands are for manually entered commands in a shell window.  If you are authoring a script, readability takes precedence over reducing keystrokes.  Scripts should always use the long names. I wish there was a better way to evangelize this principle.